### ISSUE TITLE
`buildah commit` requires an image name but man page says it's optional

### ISSUE THREAD

**Issue by TomSweeneyRedHat** [projectatomic/buildah#553]
**Description**

The `buildah commit` man page states that the image name is not a required parameter when it is.

**Steps to reproduce the issue:**
1.  buildah from alpine
2. buildah conifig --author="Tom Sweeney"
3.  buildah commit {containersId from `buildah containers`}

**Describe the results you received:**
```
# buildah commit a07bd79673d9
an image name must be specified
```

**Describe the results you expected:**
Based on the man page, this should have worked.  Perhaps the man page is incorrect?

**Output of `buildah version`:**

```
# buildah version
Version:       0.15
Go Version:    go1.9.4
Image Spec:    1.0.0
Runtime Spec:  1.0.0
Git Commit:    fb367e4
Built:         Thu Mar 29 19:59:01 2018
OS/Arch:       linux/amd64

```

**Output of `cat /etc/*release`:**

```
# cat /etc/*release
Fedora release 27 (Twenty Seven)
NAME=Fedora
VERSION="27 (Server Edition)"
ID=fedora
VERSION_ID=27
PRETTY_NAME="Fedora 27 (Server Edition)"
ANSI_COLOR="0;34"
CPE_NAME="cpe:/o:fedoraproject:fedora:27"
HOME_URL="https://fedoraproject.org/"
SUPPORT_URL="https://fedoraproject.org/wiki/Communicating_and_getting_help"
BUG_REPORT_URL="https://bugzilla.redhat.com/"
REDHAT_BUGZILLA_PRODUCT="Fedora"
REDHAT_BUGZILLA_PRODUCT_VERSION=27
REDHAT_SUPPORT_PRODUCT="Fedora"
REDHAT_SUPPORT_PRODUCT_VERSION=27
PRIVACY_POLICY_URL="https://fedoraproject.org/wiki/Legal:PrivacyPolicy"
VARIANT="Server Edition"
VARIANT_ID=server
Fedora release 27 (Twenty Seven)
Fedora release 27 (Twenty Seven)

```

**Output of `uname -a`:**

```
# uname -a
Linux localhost.localdomain 4.15.13-300.fc27.x86_64 #1 SMP Mon Mar 26 19:06:57 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by rhatdan** [comment]
Just open a PR to fix the man page.

**Comment by TomSweeneyRedHat** [comment]
We'd a longish discussion in #podman today, docker does a commit without the name as the man page is now.  I was thinking me might want to correct it now, or possible wait and then use @baude commit that he's putting together that Buildah and Podman will both use.  That commit is planned to not require an image name.

**Comment by pixdrift** [comment]
What's the expected behaviour when the image name isn't provided? Does it use the hash or container name for the image name?

**Comment by TomSweeneyRedHat** [comment]
@pixdrift current expected behaviour is that you'll get an error telling you that you need to provide an image name.  Some time in the near future (weeks if not less) you should end up with a "<None>" image name if you don't provide one.  We're working on the fix now and I should make an issue.

**Comment by TomSweeneyRedHat** [comment]
Fixed by #554
